nncf/quantization/algorithms/min_max/backend.py

```diff
@@ -57,6 +57,13 @@ def shapeof_metatypes(self) -> List[OperatorMetatype]:
         Property for the backend-specific ShapeOf metatypes.
         """
 
+    @property
+    @abstractmethod
+    def similar_input_metatypes(self) -> List[OperatorMetatype]:
```

Comment:
Is it better than using input_metatypes which returns all possible meta types that are inputs?

Reply:
What do you mean by `input_metatypes`? `ReadValues` are not inputs themselves and are followed by `Constant` nodes every time.

Reply:
Ok, I see. Just the name `similar_input_metatypes` confused me. 
I see that this is applicable only for OpenVINO, while it could be extended to be applicable to all backends if this function will return all metatypes that should be treatable as inputs.

Reply:
Yeah, it may be extended for any type that is needed. I've tried to present a more general solution under the conditions that metatypes are not common.